lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -87,8 +87,10 @@ $executable = Get-AnsibleParam -obj $params -name "executable" -type "str" -alia
 $frequency = Get-AnsibleParam -obj $params -name "frequency" -type "str" -validateset "once","daily","weekly" -failifempty $present
 $time = Get-AnsibleParam -obj $params -name "time" -type "str" -failifempty $present
 
-# TODO: We should default to the current user
-$user = Get-AnsibleParam -obj $params -name "user" -type "str" -failifempty $present
+$user = Get-AnsibleParam -obj $params -name "user" -default "$env:USERDOMAIN\$env:USERNAME" -type "str"
+$password = Get-AnsibleParam -obj $params -name "password" -default $null -type "str"
+$runlevel = Get-AnsibleParam -obj $params -name "runlevel" -default "limited" -type "str" -validateset "limited", "highest"
+$do_not_store_password = Get-AnsibleParam -obj $params -name "do_not_store_password" -default $false -type "bool"
```

Comment:
Negative booleans are always confusing, why don't we use `store_password:` and set it to `$true` by default ?